eth/downloader/downloader.go

```diff
@@ -587,41 +587,87 @@ func (d *Downloader) fetchHeight(p *peerConnection) (*types.Header, error) {
 	}
 }
 
+// calcRequestSpan calculates what headers to request from a peer when trying to determine the
```

Comment:
The name of the method in the doc doesn't match it's actual name. Please also document what the return parameters are. They are 4 ints, which is very non obvious as to what means what.